doc/user/ospfd.rst

```diff
@@ -853,6 +853,48 @@ Redistribution
 .. index:: no router zebra
 .. clicmd:: no router zebra
 
+Graceful Restart Helper
+=======================
+
+.. index:: graceful-restart helper-only [A.B.C.D]
+.. clicmd:: graceful-restart helper-only [A.B.C.D]
+
+.. index:: no graceful-restart helper-only [A.B.C.D]
+.. clicmd:: no graceful-restart helper-only [A.B.C.D]
```

Comment:
If GR(RESTARTER) functionality is added , user can provide 'graceful-restart' so it will support both RESTARTER and HELPER functionalities. if someone wanted to enable only HELPER role , then helper_only keyword has to be given .
So for me here , helper_only looks more suitable .

Reply:
Ah ok, I thought enabling GR (restarter) is just supposed to be `[no] graceful-restart` and that's it. I think Juniper and Cisco are also doing it this way.

Reply:
@GalaxyGorilla 
Is this cli look fine ? or still you want me to modify to "helper " instead of "helper_only".

Reply:
For now this is fine I think :)
Let's later have a look how e.g. Cisco does this exactly and if it makes sense to follow their approach.